lib/peep.rb

```diff
@@ -0,0 +1,25 @@
+require 'pg'
+
+class Peep
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
```

Comment:
This if statement is repeated later in the file, is there a refactor you could do to make this file more DRY?